src/ucp/wireup/address.c

```diff
@@ -312,16 +319,57 @@ static void ucp_address_memcheck(ucp_context_h context, void *ptr, size_t size,
     }
 }
 
+static uint32_t ucp_address_pack_flags(uint64_t flags,
+                                       uint64_t cap_flags,
+                                       uint8_t start_bit)
+{
+    uint32_t result_flags;
+    uint32_t packed_flag;
+    uint8_t index;
+
+    ucs_assert(start_bit < 64);
```

Comment:
< 32 since start_bit is index in result which is uint32_t

Reply:
pls ignore if it's for input flags value which is uint64_t

Reply:
yes, it has to be 32
added more clear names to emphasize this